### PR TITLE
Add link to v2 project in activity feed

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

On homepage activity feed (latest payments:
- fix v1 handles not rendering
- add something for v2

## Screenshots or screen recordings

Screen Shot 2022-05-12 at 9 54 52 am

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/Landing/Payments.tsx [hunk 1]
```diff
@@ -23,13 +25,27 @@ export default function Payments() {
...
       'id',
-      { entity: 'project', keys: ['id'] },
+      { entity: 'project', keys: ['id', 'projectId', 'cv'] },
     ],
     first: 20,
     orderDirection: 'desc',
     orderBy: 'timestamp',
   })
 
+  const ProjectHandle = ({ project }: { project: Partial<Project> }) => {
```
Comment (peripheralist) [inline src/components/Landing/Payments.tsx:35]: nice pattern here!

File: src/components/Landing/Payments.tsx [hunk 2]
```diff
@@ -44,20 +60,14 @@ export default function Payments() {
                 borderBottom: '1px solid ' + colors.stroke.tertiary,
               }}
             >
+              <ProjectHandle project={e.project} />
```
Comment (peripheralist) [inline src/components/Landing/Payments.tsx:63]: minor style thing—don't we want this inside the flex div?

File: src/components/shared/ProjectVersionBadge.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { ThemeContext } from 'contexts/themeContext'
+import { useContext } from 'react'
+
+export default function ProjectVersionBadge({
```
Comment (peripheralist) [inline src/components/shared/ProjectVersionBadge.tsx:4]: niiiice

File: src/components/Landing/Payments.tsx (hunk 2 above)
Comment (aeolianeth) [reply src/components/Landing/Payments.tsx:63]: ohhh damn good catch. Yes i think we do. brb

**Comment by aeolianeth** [comment]
Thanks @peripheralist back to u 🏓

**Review by peripheralist — APPROVED** [review]

**Pull request merged**